docs/tempo/website/cli/_index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Tempo CLI"
+description: "Guide to using tempo-cli"
+keywords: ["tempo", "cli", "tempo-cli", "command line interface"]
+weight: 450
+---
+
+# Tempo CLI
+
+Tempo CLI is a separate executable that contains utility functions related to tempo.  It is not required for a working install, but may be helpful for analysis or troubleshooting.
+
+## Tempo CLI command syntax
+
+The general syntax for commands in Tempo CLI is:
+```bash
+tempo-cli command [subcommand] [options] [arguments...]
+```
+
+`--help` or `-h` displays the help for a command or subcommand.
+
+**Example:**
+```bash
+tempo-cli -h
+tempo-cli command [subcommand] -h
+```
+
+## Running Tempo CLI
+Tempo CLI is currently only available as source and a working go installation is required to build it.  It can be compiled to a native binary and executed normally, or it can be executed using the `go run` command. 
```

Comment:
```suggestion
Tempo CLI is currently available as a source. A working Go installation is required to build it. It can be compiled to a native binary and executed normally, or it can be executed using the `go run` command. 
```

Reply:
Updated as discussed